etl/steps/grapher/wb/2022-10-29/wb_gender/__init__.py

```diff
@@ -0,0 +1,90 @@
+from copy import deepcopy
+
+import pandas as pd
+from owid import catalog
+from structlog import get_logger
+
+import etl.grapher_helpers as gh
+from etl.paths import DATA_DIR
+
+log = get_logger()
+
+DATASET_GARDEN = DATA_DIR / "garden/wb/2022-10-29/wb_gender"
+
+
+def process_table(table: catalog.Table) -> catalog.Table:
+    log.info("Processing table...")
+    # Reset index
+    table = table.reset_index()
+    # Harmonize country names
```

Comment:
I suppose this comment is misplaced. You could actually do everything in one line: `table = table.reset_index().set_index(["country", "year"])`.